### ISSUE TITLE
RSpec version detection is broken

### ISSUE THREAD

**Issue by codener** [makandra/rspec_candy#5]
The RSpec version detection method has [recently been refactored](https://github.com/makandra/rspec_candy/blame/master/lib/rspec_candy/switcher.rb). Unfortunately, my application has a `Spec` constant defined somewhere, which causes the `Switcher` class to spuriously believe RSpec 1 is around, consequently making calls to `Spec::Example` fail.

**Comment by pouellet** [comment]
I ran into the same issue while trying to update the rspec_candy. The cucumber gem defines the `Spec` module which result in the wrong version of rspec getting detected by rspec_candy. The following seems to work well for me, I can do a Pull if you agree on the change:

```ruby
def rspec_version
  @rspec_version ||=
    begin
      require 'rspec/version'
      RSpec::Version::STRING.to_i
    rescue LoadError
      defined?(Spec) ? 1 : raise 'Cannot determine RSpec version'
    end
end
```

**Comment by henning-koch** [comment]
You're very welcome to send a PR!

**Comment by kratob** [comment]
Closed by #6.
